spec/compiler/codegen/closure_spec.cr

```diff
@@ -700,6 +700,8 @@ describe "Code gen: closure" do
 
   it "allows passing an external function along" do
     codegen(%(
+      require "prelude"
+
```

Comment:
I think if you load the prelude you don't need to `raise` (and subsequently `LibC`)  definition. I just pasted my minimal bug trigger as spec that I used to study  the generated LLVM IR.

Reply:
Oooh... I see. Makes sense. I'll remove the `raise`. I think `raise` is needed now because there will be a closure check in `LibA.a(a)`.

Reply:
It was before, that's why I added it :D

Reply:
Btw there's a funny bug if you leave it and the prelude of, the closure check generates an empty branch (label?) causing the LLVM IR verification to fail.